### PR TITLE
Remove ancient platforms support

### PR DESCRIPTION
**Pull request by undef21**
Remove WinCE and Symbian support.

### CODE REVIEW

File: fheroes2.key [hunk 1]
```diff
@@ -65,13 +65,6 @@
 # show icons		- (default keycode: 53 = '5')
 #
 # 2.4 system events:
-# emulate mouse toggle	- (default keycode: no set)
```
Comment (ihhub) [inline fheroes2.key:68]: I think we're removing the code related to mobile platforms as well. Please correct me if I'm wrong.

Comment (undef21) [reply fheroes2.key:68]: Not quite. We preserve android (if someone'll want add android support:)) code and remove WinCE and Symbian. Emulate mouse functions is enabled by WITHOUT_MOUSE macro. This macro is defined only in Makefile.mingw32ce. I think this macro was written for nonsensor screen WinCE devices, which move cursor by physical keys.

**Comment by ihhub** [comment]
Hi @Northfear could you please take a look into changes in this pull request and tell if we're safe to do this?

**Comment by Northfear** [comment]
@ihhub looks good to me

**Comment by ihhub** [comment]
@undef21 thanks a lot for this code cleanup!

**Pull request merged**